### DISCUSSION TITLE
关于Windows Dev不久前遇到的问题

### DISCUSSION THREAD

**Discussion by wudiwudidream** [Meloong-Git/PCL#4460] [提问] [answered]
打开启动器会明显出现卡了一下出现两个的问题（通常是双击打开文件会这样），运行游戏之后会出现计算内存暴涨（跟Windows计算的不符），随后是游戏崩溃，有个日志文件在桌面，不过删除了，处理器：AMD Ryzen 5 4600U with Radeon Graphics 2.10 GHz，操作系统版本26120.1340，Windows 11 专业工作站版，版本号：24H2

**Comment by wudiwudidream** [comment]
这个还有个问题，因为不知道是什么莫名原因，AMD会黑屏并报错（7月20日报到现在两次）

**Comment by MoYuan-CN** [comment, marked as answer]
转 #4461
